Description: Small fixes to spro documentation
Author: Giulio Paci <giuliopaci@gmail.com>
Forwarded: no
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: b/doc/spro.texi
===================================================================
--- a/doc/spro.texi
+++ b/doc/spro.texi
@@ -11,6 +11,13 @@
 @syncodeindex fn cp
 @syncodeindex pg cp
 
+@dircategory Development
+@dircategory Sound
+
+@direntry
+* spro: (spro).                SPro signal processing toolkit
+@end direntry
+
 @ifinfo
 SPro: Speech Signal Processing Toolkit.
 
